end-to-end-test/shared/wdio/getScreenshotName.js

```diff
@@ -8,11 +8,15 @@ module.exports = function getScreenshotName(basePath, browserName) {
         var browserWidth = browserViewport.width;
         var browserHeight = browserViewport.height;
 
+        // regular expression that matches invalid path characters for some systems (such as Windows)
```

Comment:
@pappde i'm realizing not, this will result in discord between the test label and the filename and report.   kinda minor but would rather just update the labels and the perhaps this code could throw an error when it encounters an invalid char?

Reply:
@alisman Originally I had thought that's what the code should do, but was really hesitant to introduce an error, particularly if it may not be immediately intuitive how to fix it. Also, there was precedent where existing files were corrected with underscore replacement.

FWIW, the original code already has the disconnect between test name and filename with the two substitutions (toLowerCase and replace space). Adding another substitution rule does technically increase the possibility, though unlikely, of a collision between two tests with different names that resolve to the same filename.  Using something like URL encoding may be more robust.

Let me know what you think: change to throw error, change to URL encoding, or leave it.

Reply:
👍